src/libs/actions/User.js

```diff
@@ -290,6 +259,62 @@ function clearContactMethodErrors(contactMethod, fieldName) {
     });
 }
 
+/**
+ * Adds a secondary login to a user's account
+ *
+ * @param {String} contactMethod
+ * @param {String} password
+ */
+function addNewContactMethodAndNavigate(contactMethod, password) {
+
```

Comment:
`Block must not be padded by blank lines`

Sorry I've always wanted to feel what it was like to be a linter... if even just for a fleeting moment.

Reply:
Thanks!